benchmarks/span/scenario.py

```diff
@@ -0,0 +1,70 @@
+import pyperf
+from util import gen_metrics
+from util import gen_spans
+from util import gen_tags
+
+from ddtrace import Span
+
+
+def time_start_span(loops):
+    range_it = range(loops)
+    t0 = pyperf.perf_counter()
+    for _ in range_it:
+        for _ in range(0, variant.get("nspans")):
```

Comment:
variant here is defined because it leaks from the scope below...? 🤔 we should probably pass it as an argument here

Reply:
ah good catch